example/demo/src/home/ui.tsx

```diff
@@ -0,0 +1,30 @@
+import { navigate } from '@rise-tools/kit-react-navigation/server'
+import { Button, YStack } from '@rise-tools/kitchen-sink/server'
+
+export const models = {
+  '': Home,
```

Comment:
this feels strange. instead I think we should `export function Home` and assign the empty string to Home inside of `src/index`